src/Kernels.h

```diff
@@ -1,6 +1,18 @@
 #ifndef COVIDSIM_KERNELS_H_INCLUDED_
 #define COVIDSIM_KERNELS_H_INCLUDED_
 
+enum KernelType {
```

Comment:
```suggestion
enum class KernelType {
```
Since this is a C++11 codebase, the use of enum classes should be preferred. Casts to / from int will need to be made explicit, but ultimately leads to fewer surprises in the long-run.

Reply:
Done